capstone360/setup_mockup_db.py

```diff
@@ -7,6 +7,13 @@
 
 
 import sqlite3
+from cryptography.fernet import Fernet
+
+# Get encryption key.
+key_file = open("key.txt")
```

Comment:
You can use a "with" statement here if you want. See https://stackoverflow.com/questions/5773545/what-does-python-3-2-with-as-do (the second response)